lib/rubocop/cop/rspec/repeated_example.rb

```diff
@@ -0,0 +1,41 @@
+module RuboCop
+  module Cop
+    module RSpec
+      # Check for repeated examples within example groups.
+      #
+      # @example
+      #
+      #    it 'is valid' do
+      #      expect(user).to be_valid
+      #    end
+      #
+      #    it 'is valid' do
```

Comment:
If we instead use two different descriptions, I think it will be more clear what this cop is checking.